project_event/tests/test_project_event_task.py

```diff
@@ -215,18 +236,42 @@ def test_090_create_orphan_task(self):
             'partner_id': self.project_1.partner_id.id,
             'room_id': self.room_1.id,
             'date_start': fields.Datetime.to_string(datetime.today()),
-            'date_end': fields.Datetime.to_string(datetime.today() + \
-                        timedelta(hours=4)),
-            }
+            'date_end': fields.Datetime.to_string(datetime.today() +
+                                                  timedelta(hours=4)),
+        }
         orphan_task = self.Tasks.create(vals)
         self.assertEqual(
             orphan_task.name,
             'Orphan task'
         )
-    
+
     def test_100_compute_complete_name_activity(self):
-        self.assertEqual(self.activity_1.name,'Test Activity 1')
+        self.assertEqual(self.activity_1.name, 'Test Activity 1')
 
     def test_110_compute_complete_name_task(self):
         complete_name = '%s / %s' % (self.task_2.code, self.task_2.name)
-        self.assertEqual(self.task_2.complete_name,complete_name)
+        self.assertEqual(self.task_2.complete_name, complete_name)
+
+    def test_120_onchange_parent_id(self):
+        self.task_1.parent_id = self.activity_2
```

Comment:
activity_1 and activity_2 have the same responsible you must change the responsible of the activity_2 for example to another responsible (responsible_2 : you need to add it if it doesn't exist) and check if the responsible of the task is changed to the responsible2 when we change the parent from activity_1 to activity_2

Reply:
@rimbendhaou :Ok Changed as per your comments but the implemented functionality and corresponding test cases are completely different.